docs/developer-docs/latest/guides/process-manager.md

```diff
@@ -50,6 +56,20 @@ const strapi = require('@strapi/strapi');
 strapi().start();
 ```
 
+</code-block>
+<code-block title="TYPESCRIPT">
+
+```js
+// path: `./server.js`
+
+const strapi = require("@strapi/strapi");
+
+strapi.compile().then((appContext) => strapi(appContext).start());
```

Comment:
I think we should use the simpler way of starting the app (by only providing the dist param to the strapi constructor). People will mainly use this to start their apps, not build plugins.

Reply:
I assume you mean the 2nd example from the GH issue? If so, updated.